### PR TITLE
Implement settings close button hover

### LINKED ISSUE #439: "X" in top right corner of Settings page doesn't have an on hover effect
I confirm (by marking "x" in the [ ] below): 

- [X] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [X] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [X] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md). 

---

**Summary**
"X" in top right corner of Settings page doesn't have an on hover effect

**Steps to reproduce**

Windows 10; Mattermost Server 3.6; Mattermost Desktop 3.5
1) Go to File > Settings
2) Hover over the "X" in top right corner

**Expected behavior**
On hover effect following [Mattermost UX guidelines](https://docs.mattermost.com/developer/fx-guidelines.html#display-button-states).

This can be similar to the on hover effect of the "+" icon in desktop server tab bar.

**Observed behavior**

No on hover effect

### PR DESCRIPTION
**Pull request by wvds**
Closes #439

Before submitting, please confirm you've
 - [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
 - [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
 - [x] executed `npm run lint:js` for proper code formatting

Please provide the following information:

**Summary**
Added hover color on settings page close button.

**Issue link**
#439

### CODE REVIEW

File: src/browser/css/settings.css [hunk 1]
```diff
@@ -1,3 +1,6 @@
+.CloseButton:hover span {
+  color: #333;
```
Comment (yuya-oc) [inline src/browser/css/settings.css]: @wvds Just curious, why did you choose `#333`?

@jasonblais What color should we use here?

hover_close_333

**Comment by yuya-oc** [comment]
https://circleci.com/gh/mattermost/desktop/1057#artifacts

File: src/browser/css/settings.css (hunk 1 above)
Comment (yuya-oc) [reply src/browser/css/settings.css]: Sorry, I forgot to read your comment at https://github.com/mattermost/desktop/issues/439#issuecomment-334969769

The color is same as TabBar's add button https://github.com/mattermost/desktop/blob/446af5ef7620af6157ab4d9149a3792917d6527f/src/browser/css/components/TabBar.css#L22

**Comment by jasonblais** [comment]
@yuya-oc This is ready to merge

**Pull request merged**